src/components/drag-layer.js

```diff
@@ -66,6 +67,8 @@ class DragLayer extends React.Component {
         return <SelectionDragPreview {...props} items={[item]}/>
       case DND.LIST:
         return <ListDragPreview list={item}/>
+      case DND.FIELD:
+        return <FieldDragPreview {...props} items={[item]}/>
```

Comment:
If we pass down the drag item in a new array, we create a new array for every update (the selection/photo case above is more like a placeholder, because we will need to support multiple items eventually). Also the field drag item is, in a way, a collection item already (with mixed values etc.). So let's pass this down as `item={item}` (or better`field={item}`).